includes/tracks/class-wc-site-tracking.php

```diff
@@ -52,14 +52,49 @@ public static function is_tracking_enabled() {
 		return true;
 	}
 
+	/**
+	 * Add scripts required to record events from javascript.
+	 */
+	public static function enqueue_scripts() {
+
+		// Add w.js to the page.
+		wp_enqueue_script( 'woo-tracks', '//stats.wp.com/w.js', array(), gmdate( 'YW' ), true );
```

Comment:
We probably can go with HTTPS here.
I'm getting the follow warning on my console:

![screenshot from 2019-02-20 16-46-05](https://user-images.githubusercontent.com/1264099/53119809-0d09ec80-352f-11e9-87c0-4842eb3c9650.png)

Reply:
switched to https in https://github.com/woocommerce/woocommerce/pull/22746/commits/c655bef951335c44575ca2e7a9413770d29a1a78